src/query/storages/fuse/src/fuse_table.rs

```diff
@@ -454,9 +454,10 @@ impl Table for FuseTable {
     async fn compact(
         &self,
         ctx: Arc<dyn TableContext>,
+        segments_only: bool,
```

Comment:
Use `enum`?

Reply:
yeah, should use enum here. 
will be addressed while merging with PR #8205